lib/curlx/strparse.c

```diff
@@ -103,6 +103,11 @@ int curlx_str_quotedword(const char **linep, struct Curl_str *out,
     return STRE_BEGQUOTE;
   s++;
   while(*s && (*s != '\"')) {
+    if(*s == '\\' && s[1]) {
```

Comment:
Since this introduces support for escaping, the comment above this function needs clarification as well, doesn't it?

Reply:
test_07_70_put_earlydata[h3-limited-body] has been flaky lately so it's unlikely to be a fault of this PR.